prod-deploy/kv-mall-manifest/kv-mall.yaml

```
@@ -351,13 +353,21 @@ spec:
         app: ads
     spec:
       containers:
-        - name: ads
-          image: registry.odigos.io/kv-mall-ads:v0.3
-          ports:
-            - containerPort: 8080
-          env:
-            - name: OTEL_PYTHON_DISABLED_INSTRUMENTATIONS
-              value: sqlalchemy,pymysql
+      - name: ads
+        image: dev/ads:dev
```

Note the dev tag on production deployment:
```suggestion
        image: registry.odigos.io/kv-mall-ads:v0.xx
```